plugins/gtav/Game.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020-2021 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef GTAV_GAME
+#define GTAV_GAME
+
+#include "ProcessWindows.h"
+#include "PluginComponents_v_1_0_x.h"
+
+class Game {
+public:
+	Game(const procptr_t id, const std::string name);
```

Comment:
```suggestion
	Game(const procid_t id, const std::string &name);
```

Or:

```suggestion
	Game(const uint64_t_t id, const std::string &name);
```

Reply:
Settled for the first option. BTW the equivalent constructor of the AmongUS Game class does also not have a std::string reference. But doesn't really matter that much due to SSO.